docs/user-guide.md

```diff
@@ -66,7 +66,7 @@ The Liberty tool window provides a context menu for Liberty projects. You can ch
 
 ### Use the Go to Action IntelliJ IDEA menu
 
-The Liberty Tools plugin provides a set of actions for the **Go to Action** IntelliJ IDEA menu. You can access the **Go to Action** IntelliJ IDEA menu either by pressing **Shift** twice, then selecting the **Actions** tab, or  by pressing **Ctl/Cmd** + **Shift** + **A**. All Liberty Tools actions that are described in the following sections are also available from the **Go to Action** IntelliJ IDEA menu.
+The Liberty Tools plugin provides a set of actions for the **Go to Action** IntelliJ IDEA menu. You can access the **Go to Action** IntelliJ IDEA menu either by pressing <kbd>Shift</kbd> twice, then selecting the **Actions** tab, or  by pressing <kbd>Ctl/Cmd</kbd> + <kbd>Shift</kbd> + <kbd>A</kbd>. All Liberty Tools actions that are described in the following sections are also available from the **Go to Action** IntelliJ IDEA menu.
```

Comment:
I think you should break Ctl/Cmd into two like <kbd>Ctl</kbd>/<kbd>Cmd</kbd> so each word looks like a key. Also we call it Ctrl in the other document. Perhaps change this to Ctrl _or_ Cmd. Ditto for the other occurrences.

Reply:
Good idea, updated to: <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>A</kbd> / <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>A</kbd>.